cmd/lsif-go/main.go

```diff
@@ -96,6 +105,9 @@ func realMain() error {
 		Args:    os.Args[1:],
 	}
 
+	// TODO(@creachadair): Packages built with cgo may not include assembly
+	// files.  To index such a package you need to set CGO_ENABLED=0.  Consider
+	// maybe doing this explicitly, always.
```

Comment:
Oof you're a two spaces guy?

Reply:
I'm not deeply partisan about it. It's just how I learned to type.

Reply:
Same here. But I got the double spaces whipped out of me.

Reply:
The idealists who believe computers can automatically get spacing correct without that signal are refreshingly optimistic in their naïveté. Even Dr. Knuth realized it required input from the user, yet there are some communities you cannot blast off their high horses. (I've removed the spaces, since I really don't care that much especially in plain text comments—but I reserve the right to continue laughing and shaking my head at abject prescriptivists)